app/views/deploys/recent.html.erb

```diff
@@ -1,6 +1,7 @@
 <h1><%= title %></h1>
 
 <div class="timeline" ng-controller="<%= controller %>">
+  <input type=hidden  id=default_time_format value="<%= current_user.time_format %>">
```

Comment:
Yeah, looks like it does.  You see it blink for a sec on page reload…

On Thu, Mar 31, 2016 at 9:50 PM, Michael Grosser <notifications@github.com>
wrote:

> In app/views/deploys/recent.html.erb
> <https://github.com/zendesk/samson/pull/778#discussion_r58162224>:
>
> > @@ -1,6 +1,7 @@
> >  <h1><%= title %></h1>
> >
> >  <div class="timeline" ng-controller="<%= controller %>">
> > +  <input type=hidden  id=default_time_format value="<%= current_user.time_format %>">
>
> maybe checked="cachekd" ... but totally possible that this won't work out
> ... angular might want to set/unset these boxes :(
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/778/files/9a751543cac5b00534a49fde2b26e49c71e9a02a#r58162224>
>